src/modules/submissions.js

```diff
@@ -2027,162 +2072,166 @@
 
 	function handleAccept( data ) {
 		var newText = data.afchText;
-
-		AFCH.actions.movePage( afchPage.rawTitle, data.newTitle,
+		$.when( AFCH.actions.movePage( afchPage.rawTitle, data.newTitle,
 			'Publishing accepted [[Wikipedia:Articles for creation|Articles for creation]] submission',
-			{ movetalk: true } ) // Also move associated talk page if exists (e.g. `Draft_talk:`)
-			.done( function ( moveData ) {
```

Comment:
this whole block (the callback) seems to have had one level of indentation removed; please indent it by one level again, to reduce the size of the diff

Reply:
But if I indent it by one level I am getting an error in grunt build.

Reply:
Which error?

Reply:
Tab indentation error that 5 tabs are required instead of 4